examples/api/lib/material/navigation_drawer/navigation_drawer.1.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Builds an adaptive navigation widget layout. When the screen width is less than
```

Comment:
This description doesn't match the demo. It sounds like it would be a good example though.

Reply:
Oops sorry forgot to delete. Just updated